script/helperFunctions.sh

```diff
@@ -872,17 +872,10 @@ function checkRequiredVariablesInDotEnv() {
   fi
 
   local PRIVATE_KEY="$PRIVATE_KEY"
-  local RPC="ETH_NODE_URI_$(tr '[:lower:]' '[:upper:]' <<<"$NETWORK")"
+  local RPC="ETH_NODE_URI_$(tr '[:lower:]' '[:upper:]' <<<"$NETWORK" | sed s/-/_/g)"
```

Comment:
just FYI: there is a helper function to get the RPC_URL for a given network

Reply:
I just left this as is because there are quite a lot of lines like this. Better to refactor as part of another PR.